translations/monero-core_de.ts

```diff
@@ -31,22 +31,22 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="103"/>
         <source>Payment ID &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</source>
-        <translation>Zahlungs-ID &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</translation>
+        <translation>Zahlungs-ID &lt;font size=&apos;2&apos;&gt;(optional)&lt;/font&gt;</translation>
```

Comment:
Because it's not the beginning of the title or standalone, but follows other words ![bildschirmfoto 2017-03-30 um 08 27 51](https://cloud.githubusercontent.com/assets/26762624/24490417/ca8b5ed0-1522-11e7-98aa-881a61a20a7f.png)